src/storage/src/table/mem_table.rs

```diff
@@ -26,6 +25,8 @@ pub enum RowOp {
     Delete(Row),
     Update((Row, Row)),
```

Comment:
Seems the row for `Delete` and the old row for `Update` is never used now. Any consideration on this?